addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -14,7 +14,18 @@
         }
     }
 }
-
+.o_form_view {
+    // Small adjustment on Widget(sms x emojis) added to Text field.
+    .o_mail_emojis_mass_mailing ~ .o_mail_add_emoji{
+        margin-top: -55px;
+    }
+}
+.o_form_view {
+    // Widget(emojis) should be hidden when field has "invisible modifier" !
+    .o_invisible_modifier + .o_mail_add_emoji{
+        display: none !important;
+    }
+}
```

Comment:
That doesn't seem specific to mass_mailing.
Add that to the mail module instead in a specific commit.

And also try to avoid using the "!important" at all costs, it's very dirty.

Reply:
It's related to the fact that emojis widget is displayed outside the Charfield, so the widget appears even when the field is invisible / removing that will require new updates on widget code.

This way the widget will get the same style in [o_invisible_modifier](https://github.com/odoo/odoo/blob/1dc3c0d1a810a5b35e98d11ae6de1fc5042146c3/addons/web/static/src/scss/views.scss#L7)

Reply:
Well, I can't argue with that.
Still, it should be added in another commit and put in the mail module since it should be the case for all fields using the emojis widget.